### DISCUSSION TITLE
Is python the best coding language ever made

### DISCUSSION THREAD

**Discussion by jooapa** [jooapa/School#4] [Polls]
Is python the best coding language ever made

**Comment by miskamero** [comment]
This poll did numbers
![badpigg](https://github.com/user-attachments/assets/03dbc46e-0377-425d-ad02-29af8ff1449b)

**Reply by jooapa** [reply]
näyttäisi siltä, että 100% ihmisistä on samaa mieltä 
![image](https://github.com/user-attachments/assets/d94a6e94-9677-4051-ba4b-b60d2b7b9b7c)
